exporters/common/src/main/java/io/opentelemetry/exporter/internal/http/HttpExporterBuilder.java

```diff
@@ -82,11 +83,16 @@ public HttpExporterBuilder<T> setCompression(String compressionMethod) {
     return this;
   }
 
-  public HttpExporterBuilder<T> addHeader(String key, String value) {
-    if (headers == null) {
-      headers = new HashMap<>();
+  public HttpExporterBuilder<T> addStaticHeader(String key, String value) {
```

Comment:
Another name change suggestion: `addConstantHeader`. It's nitpicky, but because `static` has meaning in the java language spec this should make it slightly easer to read.